### DISCUSSION TITLE
Application was accepted 9 days ago but still no access to package

### DISCUSSION THREAD

**Discussion by npst-uh** [community/community#144547] [GitHub Education]
### Hi everyone,

Hey, 
I was approoved on 05.11 for the education pack but still have no access to it..
What can I do?
![Screenshot 2024-11-14 152059](https://github.com/user-attachments/assets/f7116692-671a-40ad-bf05-c83bbebac928)

**Comment by davevad93** [comment]
Hi @npst-uh ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @npst-uh was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by npst-uh** [reply]
But is there a reason why it varies so much from person to person? A friend of mine, visiting the same university, applied a couple of day after me and got accepted and had their benefits available in 3 days while its taking way over a week for me? Or is this just random?

**Reply by davevad93** [reply]
@npst-uh , I don't know if it's random or not but I don't think so, I guess it might vary depending on the documentation submitted but I can't be totally sure. Anyway don't worry you'll soon have your benefits! 👍
